### DISCUSSION TITLE
Save multiple response examples for a single request

### DISCUSSION THREAD

**Discussion by NQuintana-LV** [hoppscotch/hoppscotch#1518] [Ideas]
This app is awesome! but the only thing I think would make it a must have, would be the ability to save the responses we get as example for a particular request. This would allow us to save possible error responses to certain params on a request, or different examples of the same request. This feature is present in Postman and is a really handy feature we would love to see implemented.
Again, this app is great!. Congratulations

**Comment by liyasthomas** [comment]
This is easily possible. We'll try to make a POC in couple of days.

**Reply by NQuintana-LV** [reply]
Awesome!

**Comment by burasuk** [comment]
What is the status of this functionality?
